test/views/CaptureWebsiteViewSpec.scala

```diff
@@ -35,15 +35,14 @@ class CaptureWebsiteViewSpec extends ViewBaseSpec {
     val continueButton = "button"
     val errorSummary = "#error-summary-heading"
     val websiteFormGroup = "#content > article > form > div:nth-child(1)"
-    val removeWebsite = "#remove-website"
```

Comment:
I think something funny's happened with the merge? This selector is used on master: https://github.com/hmrc/vat-correspondence-details-frontend/blob/master/test/views/CaptureWebsiteViewSpec.scala#L87

Reply:
That is weird - I'll add it in and also update the tests to reference website instead of email

Reply:
Have you done the delete stuff? This is technically part of another task but can keep it since it's already in.

Reply:
I added the delete link just to complete the content - it just doesn't go anywhere yet :P